### PR TITLE
Add compounds to vapor pressure batch6

### PR DESCRIPTION
**Pull request by sumitnegi7**
Don't review yet needs to be rechecked once

### CODE REVIEW

File: src/data/vapor-pressure-constants.ts [hunk 1]
```diff
@@ -1503,6 +1503,306 @@ export const DIPPRVaporPressureDictionary: Record<
...
+    C3: -8.4344,
+    C4: 4.5035e-6,
+    C5: 2,
+    minimumTemperature: 223.16,
+    maximumTemperature: 606.15,
+    pressureAtMinimumTemperature: 6.41,
+    pressureAtMaximumTemperature: 2.939e6,
+  },
+
+  'trans1,1Dimethylcyclohexane': {
```
Comment (qur786) [inline src/data/vapor-pressure-constants.ts:1642]:
```suggestion
  'trans1,2Dimethylcyclohexane': {
```

File: src/data/vapor-pressure-constants.ts [hunk 2]
```diff
@@ -1503,6 +1503,306 @@ export const DIPPRVaporPressureDictionary: Record<
...
+    C3: -6.755,
+    C4: 1.3269e-6,
+    C5: 2,
+    minimumTemperature: 274.18,
+    maximumTemperature: 766,
+    pressureAtMinimumTemperature: 3.72e-2,
+    pressureAtMaximumTemperature: 2.779e6,
+  },
+
+  DimethylSilane: {
```
Comment (qur786) [inline src/data/vapor-pressure-constants.ts:1732]:
```suggestion
  Dimethylsilane: {
```

File: src/data/vapor-pressure-constants.ts [hunk 3]
```diff
@@ -1503,6 +1503,306 @@ export const DIPPRVaporPressureDictionary: Record<
...
+    name: 'Dimethyl silane',
```
Comment (qur786) [inline src/data/vapor-pressure-constants.ts:1733]:
```suggestion
    name: 'Dimethylsilane',
```

**Review by qur786 — CHANGES_REQUESTED** [review]

**Review by qur786 — APPROVED** [review]

File: src/data/vapor-pressure-constants.ts [hunk 4]
```diff
@@ -1503,6 +1503,306 @@ export const DIPPRVaporPressureDictionary: Record<
...
+    C3: -6.425,
+    C4: 1.51e-16,
+    C5: 6,
+    minimumTemperature: 122.93,
+    maximumTemperature: 402,
+    pressureAtMinimumTemperature: 4.15e-1,
+    pressureAtMaximumTemperature: 3.561e6,
+  },
+
+  DimethylSulphide: {
```
Comment (maneetgoyal) [inline src/data/vapor-pressure-constants.ts:1747]:
```suggestion
  DimethylSulfide: {
```

File: src/data/vapor-pressure-constants.ts [hunk 5]
```diff
@@ -1503,6 +1503,306 @@ export const DIPPRVaporPressureDictionary: Record<
...
+    name: 'Dimethyl sulphide',
```
Comment (maneetgoyal) [inline src/data/vapor-pressure-constants.ts:1748]:
```suggestion
    name: 'Dimethyl sulfide',
```

**Review by maneetgoyal — APPROVED** [review]
Minor updates needed

**Pull request merged**